src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -892,6 +892,24 @@ private String createProgramGroupID(final SAMFileHeader header) {
 
         final FeatureInput<? extends Feature> featureInput = new FeatureInput<>(filePath, name);
 
+        //Add datasource to the feature manager too so that it can be queried. Setting lookahead to 0 to avoid caching.
+        //Note: we are disabling lookahead here because of windowed queries that need to "look behind" as well.
```

Comment:
In my review I suggested the opposite: remove the new overload, and use the existing overload (https://github.com/broadinstitute/gatk/pull/5134#discussion_r213755234). I've argued that this would have some advantages over the current approach.

Reply:
That works.  I'll do that.